readthedocs/settings/docker.py

```diff
@@ -0,0 +1,106 @@
+import os
```

Comment:
This file should probably be called `docker_compose.py` or something. I think it's confusing otherwise.